classes/person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
```

Comment:
> - [ ] As stated earlier, kindly use `require_relative` instead of `require`. 
> 
> ```suggestion
> require_relative 'nameable'
> require_relative 'capittilize_decorator'
> require_relative 'trimmer_decorator'
> ```